protostar/starknet_gateway/gateway_facade.py

```diff
@@ -166,12 +166,14 @@ async def deploy_via_udc(
             contract_address=call.udc.to_addr,  # type: ignore
             entry_point_selector=call.udc.selector,  # type: ignore
             calldata=call.udc.calldata,  # type: ignore
-            max_fee=max_fee or 0,
-            nonce=await self._gateway_client.get_contract_nonce(
-                call.udc.to_addr,
-            )
-            if signer
-            else None,
+            max_fee=max_fee if isinstance(max_fee, int) else 0,
```

Comment:
This is solely because i want to acknowledge that we are not missing this parameter on purpose, and that it's being used. Doesn't hurt to leave it that way, and improves readability (the reader would wonder, why leave out the `max_fee`, since we take it as an argument?).